tests/test_main.py

```diff
@@ -42,6 +42,18 @@ def test_streaming():
     assert total_rows == 2000
     assert num_batches > 1
 
+def test_timeout():
+    client = get_test_client()
+    query = """
+SELECT count(*)
+FROM eth.blocks
+WHERE number > 10000000 AND number < 10100000
```

Comment:
Try using this query
```suggestion
select * from eth.recent_blocks 
  join eth.recent_traces on eth.recent_blocks.hash = eth.recent_traces.block_hash
```

Reply:
Thanks. recent table normally don't have much data and due to caching, the first time it takes a few seconds but second time is pretty quick.

Reply:
The one I used is to parse the log with window function and order. My local test ran it 10 time continuously and all above 1 sec.